src/python/WMCore/FwkJobReport/XMLParser.py

```diff
@@ -433,24 +433,24 @@ def perfStoreHandler():
         try:
             readTotalMB = storageValues.get("Timing-%s-read-totalMegabytes" % readMethod, 0) \
                           + storageValues.get("Timing-%s-readv-totalMegabytes" % readMethod, 0)
-            readMSecs   = (storageValues.get("Timing-%s-read-totalMsecs" % readMethod, 0)\
-                           + storageValues.get("Timing-%s-readv-totalMsecs" % readMethod, 0))
+            readSecs   = ((storageValues.get("Timing-%s-read-totalMsecs" % readMethod, 0)
+                           + storageValues.get("Timing-%s-readv-totalMsecs" % readMethod, 0))) // 1000
             totalReads  = storageValues.get("Timing-%s-read-numOperations" % readMethod, 0) \
                           + storageValues.get("Timing-%s-readv-numOperations" % readMethod, 0)
-            readMaxMSec = max(storageValues.get("Timing-%s-read-maxMsecs" % readMethod, 0),
-                              storageValues.get("Timing-%s-readv-maxMsecs" % readMethod, 0))
+            readMaxSec = (max(storageValues.get("Timing-%s-read-maxMsecs" % readMethod, 0),
+                              storageValues.get("Timing-%s-readv-maxMsecs" % readMethod, 0))) // 1000
             readPercOps = storageValues.get("Timing-tstoragefile-readActual-numOperations", 0)/\
                           storageValues.get("Timing-tstoragefile-read-numOperations", 0)
             readCachOps = storageValues.get("Timing-tstoragefile-readViaCache-numSuccessfulOperations", 0)/\
                           storageValues.get("Timing-tstoragefile-read-numOperations", 0)
-            readTotalT  = 1000 * storageValues.get("Timing-tstoragefile-read-totalMSecs", 0)
+            readTotalT  = storageValues.get("Timing-tstoragefile-read-totalMSecs", 0) // 1000
```

Comment:
// is integer division if we keep the sec probably we don't want get so precise numbers.